res/enkf/es_update.py

```diff
@@ -158,3 +160,5 @@ def smootherUpdate(self, run_context):
             source_fs,
             target_fs,
         )
+        self.ert.reports = update_reports
```

Comment:
The reports should be appended and not overwritten, and  preferably linked to the current  update iteration

Reply:
Yes, this is not the best solution, but not quite sure what the best is, so open to suggestions - did consider injecting more context higher in the callstack, for example configuration and target/source fs to make it unique. What do you think?